sdk/webpubsub/azure-messaging-webpubsubservice/tests/test_smoke_async.py

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from testcase import WebpubsubPowerShellPreparer
+from testcase_async import WebpubsubTestAsync
+
+
+class WebpubsubSmokeTestAsync(WebpubsubTestAsync):
+
+    @WebpubsubPowerShellPreparer()
+    async def test_health_api_status(self, webpubsub_endpoint):
+        client = self.create_client(endpoint=webpubsub_endpoint)
+        await client.health_api.get_service_status()
+
+    @WebpubsubPowerShellPreparer()
+    async def test_webpubsub_send_to_all(self, webpubsub_endpoint):
+        client = self.create_client(endpoint=webpubsub_endpoint)
+        await client.web_pub_sub.send_to_all('Hub', {'hello': 'test_webpubsub_send_to_all'})
+
+    @WebpubsubPowerShellPreparer()
+    async def test_webpubsub_send_to_all_apim_proxy(self, webpubsub_endpoint, reverse_proxy_endpoint=None):
+        client = self.create_client(endpoint=webpubsub_endpoint, reverse_proxy_endpoint=reverse_proxy_endpoint)
```

Comment:
I think the tests for reverse proxy endpoint should show that passing in this value actually makes a difference. Right now, looking at the tests I can't be sure that `reverse_proxy_endpoint` works, because the things you're looking for in the test are the same as if it wasn't passed in

Reply:
we could distinguish it in yaml file. When `reverse_proxy_endpoint` works, the url changes.